src/App.js

```diff
@@ -15,6 +16,10 @@ function App() {
         <Route exact path="/Rank" component={Rank} />
         <Route exact path="/About" component={About} />
         <Route exact path="/Donations" component={Donations} />
+        <Clock
+          timeTillDate="03 07 2019, 13:00"
+          timeFormat="DD MM YYYY, HH:mm"
```

Comment:
Niiice! Vet ikke helt om du vet hva du har gjort her, men du har sendt inn to parametere til Clock komponenten din. Er perfekt. For da kan den gjenbrukes med andre datoer andre steder på siden :)

Reply:
Tok sabla lang tid før jeg forsto hva det var. Slet lenge med å skjønne hvordan jeg skulle gi beskjed til komponenten om hvilke dato den skulle telle ned til. Men ja, tenkte også selv at det er veldig smart, fordi da er den gjenbrukbar!